components/remerge/src/storage/bootstrap.rs

```diff
@@ -15,16 +15,19 @@
 //!   - remerge/client-id
 //!   - remerge/change-counter
 
-use super::{meta, SchemaBundle};
+use super::{bundle::ToLocalReason, meta, LocalRecord, NativeRecord, SchemaBundle};
+use crate::schema::desc::RecordSchema;
+use crate::schema::error::SchemaError;
 use crate::error::*;
 use crate::Guid;
-use rusqlite::Connection;
+use rusqlite::{Connection, named_params};
 use std::sync::Arc;
 
 pub(super) fn load_or_bootstrap(
     db: &Connection,
     native: super::NativeSchemaAndText<'_>,
 ) -> Result<(SchemaBundle, Guid)> {
+    println!("URGHHH!! {:?}", meta::try_get::<String>(db, meta::COLLECTION_NAME));
     if let Some(name) = meta::try_get::<String>(db, meta::COLLECTION_NAME)? {
```

Comment:
@thomcc Wondering what the right way is to set the collection name here as I'm getting None for COLLECTION_NAME